<launch>
  <arg name="camera_topic" default="/realsense_d435_forward/rgb/image_raw"/>
  <arg name="image_transport_mode" default="raw"/>
  <arg name="camera_info_topic" default="/realsense_d435_forward/rgb/camera_info"/>

  <!-- send footstep request to, compute the velocity to follow. execute it -->
  <node pkg="anakin_ros" type="anakin_detector_node" name="anakin_detector_node" output="screen">
    <param name="camera_topic" value="$(arg camera_topic)" />
    <param name="image_transport_mode" value="$(arg image_transport_mode)" />
    <param name="camera_info_topic" value="$(arg camera_info_topic)" />
    <param name="show_window" value="false" />
    <param name="verbose" value="false" />
    <param name="max_processing_fps" value="1" />

    <param name="tag_size" value="0.1735" />
  </node>
</launch>
